event-asynchronous/README.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: pattern
+title: Event-based Asynchronous
+folder: event-asynchronous
+permalink: /patterns/event-asynchronous/
+categories: Other
+tags:
+ - Java
```

Comment:
Can you think proper categories and tags for the pattern? For reference see http://java-design-patterns.com/patterns/